src/JustSaying/Messaging/Middleware/NoopMiddleware`2.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+
+namespace JustSaying.Messaging.Middleware
+{
+    internal class NoopMiddleware<TContext, TOut> : MiddlewareBase<TContext, TOut>
```

Comment:
Suggest based on the `NullLoggerFactory` and `NullLogger` convention of Microsoft's, we call this `NullMiddleware`?

Reply:
This was originally `InnerMiddleware` as it's just a wrapper around the actual operation.
We changed it because it sounded like something internal and originally it was public.

Reply:
I guess I'm just not a fan of `Noop`, as it's technically _"No-Op"_, which would be `NoOp`, and that looks equally weird to me.